certified-connectors/OzyApprovals/apiDefinition.swagger.json

```diff
@@ -1 +1,1285 @@
+	  },
+	  {
+		  "propertyName":"Categories",
+		  "propertyValue":"Human Resouces, Productivity"
+	  },
+  ],
+  
+  "host": "ozyapprovals.com",
+  "basePath": "/index.php/services/api/v1/rpa/common",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "parameters": {
+    "get_approval_processes": {
+      "x-ms-summary": "Approval Process",
+      "x-ms-dynamic-values": {
+        "operationId": "GetApprovalProcesses",
+        "value-collection": "msg",
+        "value-path": "id",
+        "value-title": "name"
+      },
+      "required": true,
+      "in": "path",
+      "name": "process_id",
+      "description": "Select the approval process",
+      "x-ms-visibility": "important",
+      "type": "string"
+    }
+  },
+  "paths": {
+    "/subscribe_trigger": {
+      "x-ms-notification-content": {
+        "description": "Trigger Payload",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "id": {
+              "type": "string",
+              "description": "id"
+            },
+            "ticket_id": {
+              "type": "string",
+              "description": "Ticket ID"
+            },
+            "date_created": {
+              "type": "string",
+              "description": "Date Ticket Created"
+            },
+            "request_platform": {
+              "type": "string",
+              "description": "Request Platform"
+            },
+            "process_name": {
+              "type": "string",
+              "description": "Process Name"
+            },
+            "process_description": {
+              "type": "string",
+              "description": "Process Description"
+            },
+            "process_id": {
+              "type": "string",
+              "description": "Process ID"
+            },
+            "request_id": {
+              "type": "string",
+              "description": "Request ID"
+            },
+            "stage_id": {
+              "type": "string",
+              "description": "Stage ID"
+            },
+            "stage_name": {
+              "type": "string",
+              "description": "Stage Name"
+            },
+            "stage_description": {
+              "type": "string",
+              "description": "Stage Description"
+            },
+            "actioned_by": {
+              "type": "string",
+              "description": "Actioned By"
+            },
+            "date_actioned": {
+              "type": "string",
+              "description": "Date Actioned"
+            },
+            "action_data": {
+              "type": "object"
+            }
+          }
+        },
+        "headers": {
+          "Content-Type": {
+            "description": "Content-Type",
+            "type": "string"
+          }
+        }
+      },
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
```

Comment:
please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.